resources/clj-kondo.exports/tensegritics/clojuredart/hooks/flutter.clj

```diff
@@ -0,0 +1,54 @@
+(ns hooks.flutter
+  (:require [clj-kondo.hooks-api :as api]
+            [clojure.set :as set]
+            [clojure.string :as str]))
+
+(defn widget 
+  "Hook for cljd.flutter.alpha/widget macro"
+  [{:keys [:node]}]
+    (let [args (rest (:children node))
+          opts-nodes (take-while (comp api/keyword-node? first) (partition 2 args))
+          pairs->value (fn [k] (first (filter #(-> % first (api/sexpr) (= k)) opts-nodes)))
+          [_ state] (pairs->value :state)
+          [_ context] (pairs->value :context)
+          [_ watch] (pairs->value :watch)
+          [_ key] (pairs->value :key)
+
+          bindings-count (* 2 (count opts-nodes))
+          body (drop bindings-count args)
+          error (fn [msg] (throw (ex-info msg {})))
+          unknown-keys (set/difference 
+                         (into #{} (map (comp api/sexpr first) opts-nodes))
+                         #{:state :key :watch :context})]
+
+      (cond
+        (and (seq (filter api/keyword-node? args))   ; has top keywords
```

Comment:
could raise some false positive but it's so unlikely and useless that's ok

Reply:
What examples do you think may become a false positive?
Only when there is a key-word pair, it raises an exception 
<img width="385" alt="image" src="https://user-images.githubusercontent.com/14236531/164402263-593d7724-ea3a-46e2-a125-4432d6960133.png">